src/hooks/screen/useChatScreenHook.ts

```diff
@@ -11,18 +12,31 @@ import SignedMessageRepo from '../../service/repo/signedMessageRepo';
 import UseChatScreenHook from '../../../types/hooks/screens/useChatScreenHook.types';
 import useChatUtilsHook from '../core/chat/useChatUtilsHook';
 import useLocalDatabaseHook from '../../database/hooks/useLocalDatabaseHook';
+import useMessageHook from './useMessageHook';
+import {ANONYMOUS} from '../core/constant';
 import {
   CHANNEL_TYPE_ANONYMOUS,
   CHANNEL_TYPE_GROUP,
-  CHANNEL_TYPE_PERSONAL
+  CHANNEL_TYPE_PERSONAL,
+  MESSAGE_TYPE_REGULAR,
+  MESSAGE_TYPE_REPLY
 } from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useMessageHook` and several constants are added. Ensure these new imports are used appropriately in the code.
```diff
+ import useMessageHook from './useMessageHook';
+ import {ANONYMOUS} from '../core/constant';
+  import {
+    CHANNEL_TYPE_ANONYMOUS,
+    CHANNEL_TYPE_GROUP,
+   CHANNEL_TYPE_PERSONAL,
+   MESSAGE_TYPE_REGULAR,
+   MESSAGE_TYPE_REPLY
+  } from '../../utils/constants';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->